### PR TITLE
[doc-enhance][bots][1700930]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Added note to  Per bot per thread limit section.

### CODE REVIEW

**Review by Nikitha-MSFT — DISMISSED** [review]
Reviewed and approved

File: msteams-platform/bots/how-to/rate-limit.md [hunk 1]
```diff
@@ -108,10 +108,12 @@ You can also handle rate limit using the per bot per thread limit.
 
 ## Per bot per thread limit
 
->[!NOTE]
-> Message splitting at the service level results in higher than expected requests per second (RPS). If you are concerned about approaching the limits, you must implement the [backoff strategy](#backoff-example). The values provided in this section are for estimation only.
+The per bot per thread limit controls the traffic that a bot is allowed to generate in a single conversation. A conversation is 1:1 between bot and user, a group chat, or a channel in a team. So, if the application sends one bot message to each user, it does not throttle.
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/rate-limit.md:111]: Suggested correction:
...in a team. Therefore, if the application sends...

What is not throttled? It is not coming out clearly.

File: msteams-platform/bots/how-to/rate-limit.md [hunk 2]
```diff
@@ -108,10 +108,12 @@ You can also handle rate limit using the per bot per thread limit.
...
 
-The per bot per thread limit controls the traffic that a bot is allowed to generate on a single conversation. A conversation here is 1:1 between bot and user, a group chat, or a channel in a team.
+>[!NOTE]
+> * The thread limit of 3600 seconds-1800 applies only if multiple bot messages are sent to a single user.
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/rate-limit.md:114]: What is 1800? Does it have a unit?

File: msteams-platform/bots/how-to/rate-limit.md [hunk 3]
```diff
@@ -108,10 +108,12 @@ You can also handle rate limit using the per bot per thread limit.
...
 ## Per bot per thread limit
 
->[!NOTE]
-> Message splitting at the service level results in higher than expected requests per second (RPS). If you are concerned about approaching the limits, you must implement the [backoff strategy](#backoff-example). The values provided in this section are for estimation only.
+The per bot per thread limit controls the traffic that a bot is allowed to generate in a single conversation. A conversation is 1:1 between bot and user, a group chat, or a channel in a team. So, if the application sends one bot message to each user, it does not throttle.
 
-The per bot per thread limit controls the traffic that a bot is allowed to generate on a single conversation. A conversation here is 1:1 between bot and user, a group chat, or a channel in a team.
+>[!NOTE]
+> * The thread limit of 3600 seconds-1800 applies only if multiple bot messages are sent to a single user. 
+> * The global limit per app per tenant is 30 Requests Per Second (RPS). Hence, the total number of bot messages per second must not cross the thread limit mentioned in the document.
```
Comment (KirtiPereira) [inline msteams-platform/bots/how-to/rate-limit.md:115]: Requests Per Second >> requests per second (RPS)

... Therefore, the total number...

The information is available in which document? Provide link.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Mahesh2516 - please go through the comments, validate, and add them.

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**